Essentials/test/com/earth2me/essentials/FakeServer.java

```
@@ -195,18 +196,33 @@ public int scheduleSyncDelayedTask(Plugin plugin, Runnable r, long l)
 			}
 
 			@Override
+			public int scheduleSyncDelayedTask(Plugin plugin, BukkitRunnable bukkitRunnable, long l) {
+				return 0;
```

Maybe this should <code>throw new UnsupportedOperationException("Not supported yet.");</code>